Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nginx:alpine
+
+RUN apk --no-cache add nodejs-lts git
+
+WORKDIR /tmp/app
+ADD . ./
+
+RUN npm install -g yarn bower
+RUN yarn
+RUN bower --allow-root install
+RUN yarn run build optimize-images
+RUN yarn run build
+
+COPY ./build/bundled/. /usr/share/nginx/html
```

Comment:
Dit kopieert dus niet de contents van de build die net uitgevoerd met als resultaat de `./build` naar de locatie om het te serven? Als dat zo is klopt het inderdaad nog niet.